pattern-library/sass/patterns/_tables.scss

```diff
@@ -23,3 +24,148 @@ $table-border-color:                        palette(grayscale, x-light) !default
 // ----------------------------
 // #GLOBAL
 // ----------------------------
+
+
+%highlight {
+  background-color: palette(warning, x-trans);
+}
+
+%edx-table-base {
+
+  caption {
+    @extend %copy-meta;
+    @extend %copy-de-emphasized;
+    @include text-align(left);
+    margin-bottom: spacing-vertical(x-small);
+  }
+
+  tr:hover {
+    background-color: $table-focus-bg;
+  }
+
+  th {
+    @include text-align(left);
+    font-weight: font-weight(semi-bold);
+  }
+
+  th,
+  td {
+    padding: $table-cell-padding;
+    border: 1px solid $table-border-color;
+  }
+
+  tfoot {
+
+    tr {
+      border-top: 2px solid palette(grayscale, light);
+      font-weight: font-weight(semi-bold);
+    }
+  }
+
+  // can be used on cols, rows, or cells
+  .is-highlighted {
+    @extend %highlight;
+  }
+
+  // can be used on rows or cells only
```

Comment:
Okay, I'll delete them from here, but assume the classes exist so I can use them in the tables and reference them in the doc.